OpenUpSA/wazimap-ng-ui
# Added rich data panel link in datamapper

## Description
* Added rich data panel link in datamapper
* Added GUI test
* Show link only if there is valid theme count or profile data for current geography
* Remove link panel in case of loading state
* clicking on link should open rich data view

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-381

## How is it tested automatically?
Added a GUI test

## How should a reviewer test it locally
Go to Datamapper
If there is valid data in rich data view we show a link and clicking that link should open rich data panel
If there is no data in rich data panel link should be hidden

We can test it on Profile : YE (8) and GDC (11)

## Screenshots


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
